Makefile

```diff
@@ -311,3 +314,58 @@ build-chart-ttl.sh: export OPERATOR_IMAGE_TAG = 24h
 build-chart-ttl.sh: export CHART_REMOTE = oci://ttl.sh/${CURRENT_USER}
 build-chart-ttl.sh:
 	cd charts/embedded-cluster-operator && ../../scripts/publish-helm-chart.sh
+
+.PHONY: apko-build
+apko-build: export IMAGE ?= ttl.sh/${CURRENT_USER}/embedded-cluster-operator-image:24h
+apko-build: export ARCHS ?= amd64
+apko-build: apko-template
+	docker run -v "${PWD}":/work -w /work/build \
+		cgr.dev/chainguard/apko build apko.yaml ${IMAGE} apko.tar \
+			--arch ${ARCHS}
+
+.PHONY: apko-publish
+apko-publish: export IMAGE ?= ttl.sh/${CURRENT_USER}/embedded-cluster-operator-image:24h
+apko-publish: export ARCHS ?= amd64
+apko-publish: apko-template
+	docker run -v "${PWD}":/work -w /work/build -v "${PWD}"/build/.docker:/root/.docker \
+		cgr.dev/chainguard/apko publish apko.yaml ${IMAGE} \
+			--arch ${ARCHS} | tee build/digest
+
+.PHONY: apko-login
+apko-login: check-env-REGISTRY check-env-USERNAME check-env-PASSWORD
+	docker run -v "${PWD}":/work -v "${PWD}"/build/.docker:/root/.docker -w /work/build \
+		cgr.dev/chainguard/apko login -u "${USERNAME}" \
+			--password "${PASSWORD}" "${REGISTRY}"
+
+.PHONY: melange
+melange: export ARCHS ?= amd64
+melange: melange-template
+	mkdir -p build
+	for f in pkg controllers main.go go.mod go.sum Makefile ; do \
+		rm -rf "build/$$f" && cp -r $$f build/ ; \
```

Comment:
should we add `build/` to .gitignore?

Reply:
https://github.com/replicatedhq/embedded-cluster-operator/pull/250/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947R24

Reply:
oh, totally missed it 🤦